docs/sources/get-started/bp-labels.md

```diff
@@ -1,9 +1,12 @@
 ---
-title: Best practices
+title: Label best practices
+menuTitle:  Label best practices
 description: Grafana Loki label best practices
-weight: 400
+aliases: 
+- ../best-practices/
+weight: 700
 ---
-# Best practices
+# Label best practices
 
 Grafana Loki is under active development, and we are constantly working to improve performance. But here are some of the most current best practices for labels that will give you the best experience with Loki.
```

Comment:
this paragraph is repeated verbatim in the added file above

Reply:
That's by design.